From 1fb00098a87acb3a54424b6c2476c48b80379271 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 1 Jun 2017 15:35:40 -0700
Subject: [PATCH] MANTA-3290 alarm metadata missing from MANTA-3195 commit
 Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9d9f6e3..461a7de 100644
--- a/Makefile
+++ b/Makefile
@@ -118,7 +118,8 @@ release: all deps docs $(SMF_MANIFESTS)
 	@echo "Building $(RELEASE_TARBALL)"
 	@rm -rf $(PROTO)
 	@mkdir -p $(INSTDIR)
-	cp -r   $(TOP)/bin \
+	cp -r   $(TOP)/alarm_metadata \
+		$(TOP)/bin \
 		$(TOP)/build \
 		$(TOP)/cmd \
 		$(TOP)/config \
-- 
2.21.0

